torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -262,6 +262,28 @@ auto handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyOb
   return handle_torch_function(r, nullptr, args, kwargs, torch_api, module_name);
 }
 
+auto handle_torch_function_indexing(PyObject* self, PyObject* index, PyObject* val) -> PyObject* {
+  std::string func_name = (val == nullptr) ? "__getitem__" : "__setitem__";
```

Comment:
You don't need a full std::string here, a char* is enough

Reply:
Done!